artifacts/pom.xml

```diff
@@ -146,241 +160,253 @@
+						</artifactItem>
+						<!-- artifactItem section for clamav client -->
+						<artifactItem>
+							<groupId>xyz.capybara</groupId>
+							<artifactId>clamav-client</artifactId>
+							<version>${clamav-client.version}</version>
+							<outputDirectory>${clamav-client.location}</outputDirectory>
+							<destFileName>${clamav-client.fileName}</destFileName>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for mock-sdk -->
+						<artifactItem>
+							<groupId>io.mosip.mock.sdk</groupId>
+							<artifactId>mock-sdk</artifactId>
+							<version>${mock-sdk.version}</version>
+							<outputDirectory>${mock-sdk.location}</outputDirectory>
+							<destFileName>${mock-sdk.fileName}</destFileName>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for biosdk-client -->
+						<artifactItem>
+							<groupId>io.mosip.biosdk</groupId>
+							<artifactId>biosdk-client</artifactId>
+							<version>${biosdk-client.version}</version>
+							<outputDirectory>${biosdk-client.location}</outputDirectory>
+							<destFileName>${biosdk-client.fileName}</destFileName>
+							<classifier>jar-with-dependencies</classifier>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for demosdk client -->
+						<artifactItem>
+							<groupId>io.mosip.demosdk</groupId>
+							<artifactId>demosdk</artifactId>
+							<version>${demosdk-client.version}</version>
+							<outputDirectory>${demosdk-client.location}</outputDirectory>
+							<destFileName>${demosdk-client.fileName}</destFileName>
+							<classifier>jar-with-dependencies</classifier>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for image-compressor dependency -->
+						<artifactItem>
+							<groupId>io.mosip.image.compressor</groupId>
+							<artifactId>image-compressor</artifactId>
+							<version>${image-compressor.version}</version>
+							<outputDirectory>${image-compressor.location}</outputDirectory>
+							<destFileName>${image-compressor.fileName}</destFileName>
+							<classifier>jar-with-dependencies</classifier>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for biosdk library -->
+						<!-- Commented as per discussion. as jar with dependencies will have components of jar as well.
+                                        <artifactItem>
+                                            <groupId>io.mosip.mock.sdk</groupId>
+                                            <artifactId>mock-sdk</artifactId>
+                                            <version>${mock-sdk.version}</version>
+                                             <outputDirectory>${biosdk-client-lib.location}</outputDirectory>
+                                            <destFileName>${mock-sdk.fileName}</destFileName>
+                                            <type>jar</type>
+                                        </artifactItem>	-->
```

Comment:
Commented artifact item can be removed as not needed.

Reply:
done